documentation/modules/proc-jmxtrans-deployment.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka.adoc
+// assembly-jmxtrans.adoc.
+
+[id='proc-jmxtrans-deployment-{context}']
+= Configuring a JMXTrans deployment
+
+.Prerequisites
+* A running kubernetes cluster
+
+You can configure a JmxTrans deployment using the jmxTrans property in the following resources:
+
+* `Kafka.spec`
```

Comment:
I would just say "You can configure a JmxTrans deployment by using the `Kafka.spec.jmxTrans` property. @PaulRMellor wdyt?